scalableminds/webknossos
# Create organization folder lazily

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- run wk as demo instance and insert an unreachable datastore in psql
- try to create a new organization => should work now

### Issues:
- fixes #4760 

------
